operators/admin/olm-cs-podsched.adoc

```diff
@@ -6,26 +6,48 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its spec:
+When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its specification:
 
-* Only the `kubernetes.io/os=linux` node selector
-* No priority class name
-* No tolerations
+* Only the `kubernetes.io/os=linux` node selector.
+* The default priority class name: `system-cluster-critical`.
+* No tolerations.
 
 As an administrator, you can override these values by modifying fields in the `CatalogSource` object's optional `spec.grpcPodConfig` section.
 
+[IMPORTANT]
```

Comment:
While technically accurate, this admonition should not contain a procedure [1].  I suggest keeping it as minimal as possible, perhaps only the first two sentences. You could then break out everything that begins on line 23 as a new procedure module.

[1] https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
Thanks for the review, @mjpytlak . I updated the PR based on your feedback. Please let me know your thoughts.